example/lib/main.dart

```diff
@@ -309,453 +174,221 @@ class _ExampleMapState extends State<ExampleMap> {
     return layer;
   }
 
-  Future<GraphicsLayer?> _createGraphicLayer({
-    required String layerId,
-    ElevationMode elevationMode = ElevationMode.relativeToScene,
-  }) async {
-    final layer = await _controller?.addGraphicsLayer(
-      layerId: layerId,
-      options: GraphicsLayerOptions(
-        fields: <Field>[
-          Field(name: 'oid', type: 'oid'),
-          Field(name: 'id', type: 'string'),
-          Field(name: 'family', type: 'string'),
-          Field(name: 'name', type: 'string'),
-        ],
-        featureReduction: FeatureReductionCluster().toJson(),
-        elevationMode: elevationMode,
-      ),
-    );
-    return layer;
-  }
-
-  void _removeGraphic({
-    required String layerId,
-    required String objectId,
-  }) {
-    _controller?.removeGraphic(layerId: layerId, objectId: objectId);
-  }
-
-  Future<FeatureLayer?> _createFeatureLayer({
-    required String layerId,
-  }) async {
-    final layer = await _controller?.addFeatureLayer(
-      layerId: layerId,
-      options: FeatureLayerOptions(
-        fields: <Field>[
-          Field(name: 'oid', type: 'oid'),
-          Field(name: 'id', type: 'string'),
-          Field(name: 'family', type: 'string'),
-          Field(name: 'name', type: 'string'),
-        ],
-        symbol: _markerSymbol,
-      ),
-    );
-    return layer;
-  }
-
-  void _makePolylineVisible({required List<LatLng> points}) {
-    _controller?.moveCameraToPoints(
-      points: points,
-      padding: 30,
-    );
-  }
-
-  void _addPolygon({
-    required String layerId,
-    required PolygonGraphic graphic,
-  }) {
-    _controller?.addGraphic(
-      layerId: layerId,
-      graphic: graphic,
-    );
-  }
-
-  void _connectTwoPinsWithPolyline({
-    required String id,
-    required String name,
-    required LatLng start,
-    required LatLng end,
-  }) {
-    _controller?.addGraphic(
-      layerId: _lineLayerId,
-      graphic: PolylineGraphic(
-        paths: [
-          [
-            [start.longitude, start.latitude, 10.0],
-            [end.longitude, end.latitude, 10.0],
-          ]
-        ],
-        symbol: const SimpleLineSymbol(
-          color: Colors.purple,
-          style: PolylineStyle.shortDashDotDot,
-          width: 3,
-          marker: LineSymbolMarker(
-            color: Colors.green,
-            colorOpacity: 1,
-            style: MarkerStyle.circle,
-          ),
-        ),
-        attributes: Attributes({'id': id, 'name': name}),
-      ),
-    );
-  }
-
   @override
   Widget build(BuildContext context) {
+    final isLargeScreen = MediaQuery.sizeOf(context).width > 800;
     return Scaffold(
       key: _scaffoldKey,
-      body: Stack(
+      drawer: isLargeScreen
+          ? null
+          : NavigationDrawer(children: _buildTiles(context)),
```

Comment:
Then resizing the window on web you can see that the navbar can be opened but its already visible:


https://github.com/user-attachments/assets/e07ffc7d-3f9f-4e58-8f24-c96890826f4d